### PR TITLE
Polish 2.5.0 Changelog

### PR DESCRIPTION
**Pull request by daschuer**
I have reorganized it a bit and removed refactoring PRs that are not of interest for a regular user. 
This shall match earlier enties where we have the same level of details.

### CODE REVIEW

File: res/linux/org.mixxx.Mixxx.metainfo.xml [hunk 1]
```diff
@@ -657,331 +681,113 @@
...
-   #13904
-  </li>
- </ul>
- <p>
-  Experimental iOs support
- </p>
- <ul>
-  <li>
-   CMakeLists: Support building for iOS
+   iOs support: Mixxx can be build for iOs
```
Comment (ywwg) [inline res/linux/org.mixxx.Mixxx.metainfo.xml:759]: apple writes it this way:

```suggestion
   iOS support: Mixxx can be build for iOS
```

**Review by ywwg — APPROVED** [review]
this looks great! I didn't find any grammar issues to make comments about and it describes the new stuff well.

**Comment by daschuer** [comment]
Done

**Comment by ronso0** [comment]
LGTM too!

**Review by ronso0 — APPROVED** [review]

**Comment by acolombier** [comment]
LGTM! Thank you!

**Pull request merged**